### ISSUE TITLE
Revision -- Feedback 3

### ISSUE THREAD

**Issue by apoorvagnihotri** [distillpub/post--bayesian-optimization#6]
- [x] learning rate description says — all real numbers. That doesn’t make sense. Isn’t this on a log scale also?
- [x] for hyperparameter for Ridge,  mention the equation showing W, c, X, and y, and also add that a GD based approach would also have a learning rate/step size hyperparameter
- [x] Comparison of different acquisition functions on gold mining figure — legend should be EI-PI and GP-UCB. Also, for all plots showing random stdev in comparison, make the random alpha lower (more transparent)
- [x] Last plot (YLabel has achieved misspelled). Just remove the title intact for this plot. 
- [x] to put the neural network computation into perspective: add some numbers. How much time each evaluation (each hyperparameter combination)  took on your laptop and what was the final accuracy achieved and what were the final hyper parameters learnt. Also, then mention that you could save a lot of time given the heavy computational requirements if you needed grid search instead. 
- [x] modify the readme with the correct title and the author information, Also link the IPYNB that was used for the experiments. 
- [x] for the initial submission, remove the smileys.
- [x] Run the entire text through grammarly once.
- [x] See if we can add additional citations.
- [x] In further reading mention Parallel BO.
- [ ] Link some talks from GP summer schools on Bayesian Optimization also.
- [ ] Keep on reading the post slowly multiple times to see if there is any improvement we can make.
- [x] citations don't appear correctly - like #17, #8 and #3 citation.
